lib/slack/ApiMock.js

```diff
@@ -23,10 +23,22 @@ module.exports.bindMockApi = function (controller) {
     const apiLogByKey = new Logger();
     const httpBodyLog = [];
 
-    /*
-    * monkey patch of axios mock adapter
-    * axios mock does not return request and headers objects for response
-    * */
+	/*
+	* workaround to prevent fail for slack api initialization
+	* https://github.com/howdyai/botkit/blob/v4.6.1/packages/botbuilder-adapter-slack/src/slack_adapter.ts#L128-L137
+	* */
+	process.exit = () => {};
```

Comment:
What's going on here?

Reply:
@ihorrusinko and I talked about this. he's doing it because botkit tries to ping slack api immediately in the constructor so we cannot mock slack api before the constructor is built. Is there a better way we could do this?

Reply:
I've removed it. 
We use full botkit initialisation instead only bot version
https://github.com/gratifyguy/botkit-mock/pull/57/files#diff-08a4a90fec7ec7a039fb74fba5054ad2R82-R92

it also simplified initialisation of slack adapter  
https://github.com/gratifyguy/botkit-mock/pull/57/files#diff-e8bf6f445c0d957ef2fc5a81bb1851cdR15

Reply:
Awesome work here Ihor.